client/src/components/Logs.tsx

```diff
@@ -163,14 +166,14 @@ const LogDownloadButton = ({
   return (
     <Button
       data-cy="session-log-download-button"
-      color={color ?? "rk-green"}
+      color={color ?? "primary"}
       size={size ?? "s"}
       disabled={!canDownload(logs)}
       onClick={() => {
         save();
       }}
     >
-      <FontAwesomeIcon icon={faSave} />
+      <FileEarmarkArrowDown className={cx("me-2", "text-icon")} />
```

Comment:
We should settle on `me-1` and `ms-1` for icons. This will make it consistent across all uses of icons next to text.

```suggestion
      <FileEarmarkArrowDown className={cx("me-1", "text-icon")} />
```

Reply:
I actually changed all the occurrences I found in 2.0 components to `me-2` or `ms-2` for consistency :facepalm: 
I'm not sure which looks best -- `2` feels better on headers and `1` on buttons or links.
I'm ok with both solutions, but changing it to `1` is quite some work. Should we quickly touch base with the whole team in the next standup and do it in the next round? I also have a different proposal to prevent accidentally showing additional margins on smaller window widths.

Reply:
I changed it to always use a space of `1` to maintain consistency everywhere.
I think using whitespaces would be best, but I don't want to include that here -- we can discuss it and see if it makes sense to create a separate issue